src/utils/responseHandlers.js

```diff
@@ -3,12 +3,14 @@
  * @param {integer} code
  * @param {string} message
  * @param {string} token
+ * @param {string} data
  * @returns {object} response
  * @description Returns a successful response
  */
-const successResponse = async (res, code, message, token) => res.status(code).json({
+const successResponse = (res, code, message, token, data = null) => res.status(code).json({
```

Comment:
Can you use the `...rest` operator when passing additional parameters to res, code, and message?

Reply:
that'd be good, but for this function I had to specify exactly which param I need to hold res data